app/Http/Controllers/BookController.php

```
@@ -30,4 +31,43 @@ public function getBookFromId(int $id)
             'message' => 'Book successfully found',
         ]);
     }
+
+    public function claimBook(int $id, Request $request)
+    {
+
+        $bookToUpdate = Book::find($id);
+
+        if ($bookToUpdate) {
```

You should negate this if statement - if you make it so its checking if there is not a book to update, you can fail early which is always preferable.

If no book to update, return the response (not found 404 one). It should help you simplify the code a little bit (removing a nested if statement)